### DISCUSSION TITLE
Use darlinghq for macOS apps?

### DISCUSSION THREAD

**Discussion by superbonaci** [ravynsoft/ravynos#203] [Q&A]
Are you planning to use [darlinghq](https://www.darlinghq.org/) to run macOS apps? Or it's not necessary or will work differently?

**Comment by mszoek** [comment]
We can't use it directly since it targets Linux. We use a similar approach of integrating Mach support into the BSD kernel and providing a compatible environment (frameworks, tools, etc) for apps to run on.

**Comment by superbonaci** [comment]
Great. Darling has very limited support regarding graphical interfaces, what about your approach? Could be even better, being able to use Adobe Suite products?

**Reply by mszoek** [reply]
I think it is basically the same approach and has the same challenges - just slightly different implementation. I have not been able to run anything graphical yet.
